app/views/notifications/index.erb

```diff
@@ -56,71 +56,81 @@
           <% end %>
         </ul>
       <% end %>
-
-      <ul style="list-style-type: none; margin: 0;">
-        <% inbox.notifications.each do |notification| %>
-          <li class="alert-row">
-            <span style="padding-right: 15px;" class="fa fa-<%= notification.icon %>"></span>
-
-            <% if notification.action == 'like' %>
-              <a href="/<%= notification.actor.username %>">
+      <% notification_hash = Hash.new([]) %>
+      <% inbox.notifications.each do |notification| %>
```

Comment:
I think we can extract this logic here to a method. Usually we avoid adding logic to the views because they are harder to test and reuse. Extracting it to a method make both things easier to do. :)